tests/xtimer_msg/Makefile

```diff
@@ -0,0 +1,8 @@
+APPLICATION = xtimer_msg
+include ../Makefile.tests_common
+
+BOARD_INSUFFICIENT_RAM := stm32f0discovery
```

Comment:
> ```
>    text    data     bss     dec     hex filename
>   13900     212    5844   19956    4df4 /data/riotbuild/riotproject/tests/vtimer_msg/bin/stm32f0discovery/vtimer_msg.elf
>   14452     184    5800   20436    4fd4 /data/riotbuild/riotproject/tests/xtimer_msg/bin/stm32f0discovery/xtimer_msg.elf
> ```

BTW, the code size disadvantage results in not using xtimer API as it is
supposed to be used, but instead using timex + divisions.